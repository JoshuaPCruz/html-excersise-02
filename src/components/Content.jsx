import React from 'react';
import '../styles/components/Content.css';
import image2 from '../images/excersice 2.png';

const Content = () => {
  return (
    <div className="Content">
      <img src={image2} alt="" />
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
        doloribus labore autem? Voluptatem recusandae Lorem ipsum dolor sit
        amet, consectetur adipisicing elit. Perferendis nisi, repellat placeat
        amet consequuntur necessitatibus odio laborum quis quae quasi eligendi
      </p>
    </div>
  );
};

export default Content;