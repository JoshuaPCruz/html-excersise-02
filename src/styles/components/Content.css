.Content {
  display: flex;
  padding: 20px;
  margin-bottom: 20px;
  border-radius: 10px;
  background-color: var(--color-secondary-white);
}

.Content img {
  width: 40%;
  margin-right: 20px;
}