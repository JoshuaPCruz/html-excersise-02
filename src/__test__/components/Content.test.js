import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Content from '../../components/Content';

describe('<Content />', () => {
  const content = shallow(<Content />);
  test('Render del componente Content', () => {
    expect(content.length).toEqual(1);
  });

  test('Render de la image', () => {
    expect(content.find('p').text()).toEqual('Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat doloribus labore autem? Voluptatem recusandae Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis nisi, repellat placeat amet consequuntur necessitatibus odio laborum quis quae quasi eligendi');
  });
});

describe('Content Snapshot', () => {
  test('Comprobar la UI del componente Content', () => {
    const content = create(<Content />);
    expect(content.toJSON()).toMatchSnapshot();
  });
});