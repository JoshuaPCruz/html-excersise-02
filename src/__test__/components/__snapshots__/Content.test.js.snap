// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Snapshot Comprobar la UI del componente Content 1`] = `
<div
  className="Content"
>
  <img
    alt=""
    src={Object {}}
  />
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat doloribus labore autem? Voluptatem recusandae Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis nisi, repellat placeat amet consequuntur necessitatibus odio laborum quis quae quasi eligendi
  </p>
</div>
`;
